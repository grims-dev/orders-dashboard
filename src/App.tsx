import React from "react";
import Productlistmain from "./components/Productlistmain";
import Layout from "./components/Layout";
import { library } from "@fortawesome/fontawesome-svg-core";
import { fas } from "@fortawesome/free-solid-svg-icons";
import { faTimesCircle, faPlusCircle } from "@fortawesome/free-solid-svg-icons";

library.add(fas, faTimesCircle, faPlusCircle);

function App() {
  return (
    <Layout>
      <Productlistmain />
    </Layout>
  );
}

export default App;
