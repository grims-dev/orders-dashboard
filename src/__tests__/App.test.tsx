import { mount } from "enzyme";
import { render } from "enzyme";
import * as React from "react";
import App from "../App";

describe("Main app renders successfully", () => {
  it("renders", () => {
    const wrapper = render(<App />);
    expect(wrapper.find("p").html()).toMatch(/loading/);
  });

  it("renders snapshot of main app", () => {
    const wrapper = render(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
