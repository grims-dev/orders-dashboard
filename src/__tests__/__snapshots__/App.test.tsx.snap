// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main app renders successfully renders snapshot of main app 1`] = `
<div
  class="sc-Axmtr iwDgON"
>
  <div
    class="Toastify"
  />
  <div
    class="sc-AxhUy kSZVCr"
    data-test="statusfilter"
  >
    <button
      class="status readytotry false"
      data-status="readytotry"
    >
      Ready to Try
    </button>
    <button
      class="status ontheway false"
      data-status="ontheway"
    >
      On the Way
    </button>
    <button
      class="status inthequeue false"
      data-status="inthequeue"
    >
      In the Queue
    </button>
    <button
      class="status outofstock false"
      data-status="outofstock"
    >
      Out of Stock
    </button>
  </div>
  <p>
    Product list is loading...
  </p>
</div>
`;
